.github/workflows/build.yml

```diff
@@ -10,9 +10,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Set imagemagick version
+        id: imagemagick 
+        run: echo "::set-output name=version::$(cat ./.github/actions/imagemagick-version)"
```

Comment:
is there a reason to prefer having a file instead of just setting an environment variable at the top of this workflow?

Reply:
As this contains everything related to the version in the actions itself. So if any enhancement needed related to version we can make it in the actions itself without touching the workflow. We get everything at the same place.

Reply:
still not sure but in the end is not a game changer